### ISSUE TITLE
Server error when accessing /report admin area via mobile

### ISSUE THREAD

**Issue by jaywink** [diaspora/diaspora#5076]
As a podmin, got a report for an offensive comment. Clicked the /report link on mobile and 500 server error happens.
##

**Comment by theworldbright** [comment]
The missing :mobile format causes the following error:
```
Missing template report/index, 
application/index with {:locale=>[:en, "en"], 
:formats=>[:mobile], :variants=>[], 
:handlers=>[:erb, :builder, :raw, :ruby, :markerb, :haml]}.

Request
Parameters:
{"format"=>"mobile"}
```
Similar errors arise on mobile not only for /report but also 
- /admins/stats
- /admins/user_search

Simply duplicating their .html.haml views and renaming them to .mobile.haml seems to solve the problem for now.

**Comment by Flaburgan** [comment]
Isn't it possible to fallback to the desktop version when the mobile view is not available? We have several other issues like this one.

**Comment by AugierLe42e** [comment]
@Flaburgan : Would be better to display 404 rather than fallback on desktop view. that could cause severe problems with the JS.

**Comment by strugee** [comment]
Add `/admin_panel` to the list of admin views that cause this

**Comment by jhass** [comment]
We shouldn't make `/admin_panel` available on mobile, at least not until we decided what to do with it and configured it.

**Comment by strugee** [comment]
Makes sense, but 500 looks like something's wrong. Perhaps a basic page saying what you just said.

**Comment by SansPseudoFix** [comment]
This issue can be closed, no? Because of #7295.

**Comment by SuperTux88** [comment]
Fixed by #7295
